relay-general/src/testutils.rs

```diff
@@ -27,12 +27,8 @@ macro_rules! assert_eq_bytes_str {
                 left == right,
                 "`left == right` in line {}:\n{}\n{}",
                 line!(),
-                difference::Changeset::new("- left", "+ right", "\n"),
-                difference::Changeset::new(
-                    &String::from_utf8_lossy(left),
-                    &String::from_utf8_lossy(right),
-                    "\n"
-                )
+                pretty_hex::pretty_hex(left),
+                pretty_hex::pretty_hex(right),
```

Comment:
This loses the marker of where the differences are btw, my test cases are simple enough that I can just spot things like this so I didn't feel like implementing something better which is probably a lot of work.